components/utilities/ai.js

```diff
@@ -270,4 +270,125 @@ export const aiMoves = (combinationType, hand, validLength) => {
   } else if (combinationType === 'double sequence') {
     return doubleSequence(hand, validLength);
   }
-};
\ No newline at end of file
+};
+
+// Build Object of available plays categorized by combination type.
+// Get probabilities of hardest to play
+// Play highest value.
+const findDoubleSequences = (hand) => {
+  const args = [6, 8, 10, 12];
```

Comment:
remind me what do these values mean?

Reply:
Those args are the possible double sequence lengths. 33 44 55 | 33 44 55 66...etc